recipes/boa/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "boa" %}
+{% set version = "0.0.1.dev0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/TheSnakePit/boa/archive/{{ version }}.tar.gz
+  sha256: 534f89793dd8d7083b919d696f7d349d2c246df9b5f6763e7e7b9d34b16d1bd8
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  entry_points:
+    - conda-mambabuild = boa.cli.mambabuild:main
+    - boa = boa.cli.boa:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+    - pyyaml
+    - colorama
+  run:
+    - conda-build >=3.19.2,<4
+    - pyyaml
+    - colorama
+    - ruamel.yaml
+    - mamba >=0.4.2
+    - tabulate
+
+test:
+  imports:
+    - boa
+    - boa.cli
+  commands:
+    - boa -h
+
+about:
+  home: https://github.com/TheSnakePit/boa
+  license: BSD-3-Clause
+  license_family: BSD
```

Comment:
It doesn't really matter but this repo data is now automatically generated from SPDX identifiers
```suggestion
```